test/inductor/test_torchinductor_opinfo.py

```diff
@@ -209,6 +209,10 @@ def format_op(op):
     "geometric": {f16},
     "linalg.eigh": {f32, f64},
     "linalg.eigvalsh": {f32, f64},
+    # Unsupported: data dependent operator: aten._local_scalar_dense.default
+    ("linspace", "tensor_overload"): {b8, f16, f32, f64, i32, i64},
+    # Unsupported: data dependent operator: aten._local_scalar_dense.default
+    ("logspace", "tensor_overload"): {b8, f16, f32, f64, i32, i64},
```

Comment:
Restructured the stack. For the inductor specific tests, the new overloads seem to not regress the existing behavior since we xfail the tests here and the [tests](https://github.com/pytorch/pytorch/actions/runs/5922762848/job/16057727407) in the OpInfo adding PR are passing, indicating that the failures were correctly xfailed.

```
2023-08-25T17:57:21.1409490Z inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_linspace_tensor_overload_cuda_float16 PASSED [0.0353s] [ 41%]
2023-08-25T17:57:21.1409786Z inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_linspace_tensor_overload_cuda_float32 PASSED [0.0341s] [ 41%]
2023-08-25T17:57:21.1410088Z inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_linspace_tensor_overload_cuda_float64 PASSED [0.0354s] [ 41%]
2023-08-25T17:57:21.1410390Z inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_linspace_tensor_overload_cuda_int32 PASSED [0.0381s] [ 41%]
2023-08-25T17:57:21.1410688Z inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_linspace_tensor_overload_cuda_int64 PASSED [0.0384s] [ 41%]
```

```
2023-08-25T17:57:21.1434002Z inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_logspace_tensor_overload_cuda_float16 PASSED [0.0376s] [ 44%]
2023-08-25T17:57:21.1434346Z inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_logspace_tensor_overload_cuda_float32 PASSED [0.0360s] [ 44%]
2023-08-25T17:57:21.1434682Z inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_logspace_tensor_overload_cuda_float64 PASSED [0.0366s] [ 44%]
2023-08-25T17:57:21.1435112Z inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_logspace_tensor_overload_cuda_int32 PASSED [0.0402s] [ 44%]
2023-08-25T17:57:21.1435409Z inductor/test_torchinductor_opinfo.py::TestInductorOpInfoCUDA::test_comprehensive_logspace_tensor_overload_cuda_int64 PASSED [0.0366s] [ 44%]
```

Reply:
Resolved :)